src/main/steps/edge-case/application-submitted/content.ts

```diff
@@ -0,0 +1,37 @@
+import { TranslationFn } from '../../../app/controller/GetController';
+import { ResourceReader } from '../../../modules/resourcereader/ResourceReader';
+
+const APPLICATION_SUBMITTED_TRANSLATION_FILE = 'application-submitted';
+
+export const generateContent: TranslationFn = content => {
+  const resourceLoader = new ResourceReader();
+  resourceLoader.Loader(APPLICATION_SUBMITTED_TRANSLATION_FILE);
+  const Translations = resourceLoader.getFileContents().translations;
+
+  const en = () => {
+    const caseId = content.userCase?.id?.toString().replace(/.{4}/g, '$&-').substring(0, 19);
+    return {
+      ...Translations.en,
+      line1: `An email has been sent to ${content.userCase?.applicantEmailAddress} that explains what will happen next.`,
+      referenceNumber: `Your reference number is:<br><strong>${caseId}</strong>`,
+    };
+  };
+  const cy = () => {
+    const caseId = content.userCase?.id?.toString().replace(/.{4}/g, '$&-').substring(0, 19);
+    return {
+      ...Translations.cy,
+      line1: `An email has been sent to ${content.userCase?.applicantEmailAddress} that explains what will happen next. (in welsh)`,
```

Comment:
Can this be added in translation.json

Reply:
I tried to put in translation.json but here we are dynamically rendering some fields from userCase which is not available on custom template.njk which has been used for this page hence needed keep here, rest all fields are in translation.json